recipes/cpp_project_framework/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.11)
```

Comment:
Remove this cmake file, as the project is header-only, it does not require cmake.

Reply:
removed in latest version